tests/Application/config/sylius/1.10/packages/security.yaml

```diff
@@ -1,13 +1,14 @@
 parameters:
```

Comment:
Shouldn't this file just be in `tests/Application/config`?

Reply:
Copied from Sylius skeleton. And this would fail in <=1.9, so it has to be in 1.10 config